test/cpp/end2end/async_end2end_test.cc

```diff
@@ -858,9 +859,12 @@ class AsyncEnd2endServerTryCancelTest : public AsyncEnd2endTest {
 
     bool expected_server_cq_result = true;
     bool ignore_cq_result = false;
+    bool want_done_tag = false;
 
     if (server_try_cancel == CANCEL_BEFORE_PROCESSING) {
-      ServerTryCancel(&srv_ctx);
+      srv_ctx.TryCancel();
+      Verifier(GetParam()).Expect(11, true).Verify(cq_.get());
```

Comment:
I thought about that, but I also thought that I'd leave it alone for the following reason. This case is the one where the cancel happens before processing, so I thought I'd start over with a clean verifier after it to insure that we're not leaking any state from this part of the test to the next. Is there a strong reason to do it the way you've suggested?